src/analysis/normed_space/basic.lean

```diff
@@ -1269,60 +1319,115 @@ begin
     exact mul_le_mul_of_nonneg_right hn.1 (norm_nonneg _) }
 end
 
+end semi_normed_space
+
+section normed_space
+
+variables [normed_field α]
+variables {E : Type*} [normed_group E] [normed_space α E]
+variables {F : Type*} [normed_group F] [normed_space α F]
+
+open normed_field
+
+theorem interior_closed_ball' [semi_normed_space ℝ E] [nontrivial E] (x : E) (r : ℝ) :
```

Comment:
Why do you have a `semi_normed_space` assumption here, since you are assuming that you have `normed_group` anyway?

Reply:
Ah yes, I probably just replaced `normed_space` with `semi_normed_space` without realizing that here it doesn't make sense.